test/controllers/videos_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class VideosControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @video = videos(:one)
```

Comment:
This syntax is wrong, which is why the tests are failing. Can you look at the other controller tests and figure out the difference?

We need to load the @video from factories.rb